docs/modules/configure/pages/initializer.adoc

```diff
@@ -441,3 +441,70 @@ To customize export fields, you can subscribe to any serialize event. Every seri
   end
 ....
 
+== Content Security Policy
+Defines additional content security policies following the structure keys are the CSP directives and the values are arrays of allowed sources
+
+List of additional content security policies to be appended to the default ones
+This is useful for adding custom CSPs for external services like Here Maps, YouTube, Twitter etc.
```

Comment:
I think we can change the wording a bit to make it more clear and giving more details about what is CSP. Here's my proposal: 
 
```suggestion

We've added support for Content Security Policy (CSP), a security mechanism used by web applications to mitigate the risk of cross-site scripting (XSS) attacks and other types of code injection vulnerabilities. It is implemented through an HTTP header that instructs the browser on which sources of content, such as JavaScript, CSS, and images, are considered trustworthy and should be loaded by the web page.

Some services that are affected by the CSP is for services like Here Maps, YouTube, Twitter, etc. We allow implementers and developers to define their own services. 

For some of the features that you use, you'll need to add it to the configuration explicitly, but others (like the Meetings iframe embed feature for Online meetings), will be added automatically. 
```